src/components/grapher/Dot.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <g class="dot--dot-container">
+    <circle
+      class="dot--circle"
+      :r="radius"
+      data-test="dot--circle"
+      :stroke="strokeColor"
+      stroke-width=".3"
+      :fill="fill"
+      :fill-opacity="filled ? 1 : 0"
+    />
+    <line
+      v-if="fwSlashed"
+      class="dot--line"
+      data-test="dot--fslash"
+      :stroke="strokeColor"
+      :x1="radius * -1.1"
+      :y1="radius * 1.1"
+      :x2="radius * 1.1"
+      :y2="radius * -1.1"
+      stroke-width=".3"
+    />
+    <line
+      v-if="bwSlashed"
+      class="dot--line"
+      data-test="dot--bslash"
+      :stroke="strokeColor"
+      :x1="radius * -1.1"
+      :y1="radius * -1.1"
+      :x2="radius * 1.1"
+      :y2="radius * 1.1"
+      stroke-width=".3"
+    />
+  </g>
+</template>
+
+<script lang="ts">
+import Vue from "vue";
+import StuntSheet from "@/models/StuntSheet";
+
+/**
+ * Renders a single dot
+ */
+export default Vue.extend({
+  name: "Dot",
+  props: {
+    dotTypeIndex: Number,
+  },
+  computed: {
+    strokeColor(): string {
+      const currentSS: StuntSheet = this.$store.getters.getSelectedStuntSheet;
+      return currentSS.dotAppearances[this.dotTypeIndex].color;
+    },
+    fill(): string {
+      const currentSS: StuntSheet = this.$store.getters.getSelectedStuntSheet;
+      return currentSS.dotAppearances[this.dotTypeIndex].fill;
+    },
+    filled(): boolean {
+      const currentSS: StuntSheet = this.$store.getters.getSelectedStuntSheet;
+      return currentSS.dotAppearances[this.dotTypeIndex].filled;
+    },
+    fwSlashed(): boolean {
+      const currentSS: StuntSheet = this.$store.getters.getSelectedStuntSheet;
+      return currentSS.dotAppearances[this.dotTypeIndex].fwSlash;
+    },
+    bwSlashed(): boolean {
+      const currentSS: StuntSheet = this.$store.getters.getSelectedStuntSheet;
+      return currentSS.dotAppearances[this.dotTypeIndex].bwSlash;
+    },
```

Comment:
Since these are all so similar... how do you feel about just having 1 computed property?
```suggestion
    dotAppearance(): DotAppearance {
      const currentSS: StuntSheet = this.$store.getters.getSelectedStuntSheet;
      return currentSS.dotAppearances[this.dotTypeIndex];
    },
```

In the template we can access the properties via `dotApperance.color`, etc.

Reply:
That's a good idea!